libraries/AP_Logger/LogStructure.h

```diff
@@ -1943,7 +1945,7 @@ struct PACKED log_PSC {
     { LOG_MODE_MSG, sizeof(log_Mode), \
       "MODE", "QMBB",         "TimeUS,Mode,ModeNum,Rsn", "s---", "F---" }, \
     { LOG_RFND_MSG, sizeof(log_RFND), \
-      "RFND", "QBCBB", "TimeUS,Instance,Dist,Stat,Orient", "s#m--", "F-B--" }, \
+      "RFND", "QBCBB", "TimeUS,Instance,Dist,Snr,Stat,Orient", "s#m--", "F-B--" }, \
```

Comment:
We need to also add to "QBCBB" and the format strings. Since it's the same type as dist, can you use that as reference? Meaning find the index of Dist in each of those and add another of that character to the string.
For QBCBB --> QBCCBB

Reply:
s#m-- to s#mm--

Reply:
Done.  See next commit

Reply:
> s#m-- to s#mm--

this becomes s#m---

'm' means meters.  SNR is in decibel